src/bmi_main_lgar.cxx

```diff
@@ -155,13 +155,15 @@ int main(int argc, char *argv[])
       // write layers data to file
       fprintf(outlayer_fptr,"# Timestep = %d, %s \n", i, time[i].c_str());
       write_state(outlayer_fptr, model_state.get_model()->head);
+      delete [] soil_moisture_wetting_front;
+      delete [] soil_thickness_wetting_front;
     }
 
   }
 
   // do final mass balance
   model_state.global_mass_balance();
```

Comment:
I suggest removing this line, as global_mass_balance is called in Finalize now.

Reply:
removed in [3fab3e8](https://github.com/NOAA-OWP/LGAR-C/pull/28/commits/3fab3e8c58cfd0c329fbaf1ce504c5a3c21d9d17)